src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  --header-nav-color: #00ACDC;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+header>a{
```

Comment:
don't use combined selector and tag selector for specific styles
add class to the element you want to style and use class selector